qcodes/instrument/base.py

```diff
@@ -434,6 +434,15 @@ def snapshot_base(self, update=False):
                 snap[attr] = getattr(self, attr)
         return snap
 
+    def print_readable_snapshot(self, update=False):
```

Comment:
Should we add a docstring explaining that this returns name, value and units for all parameters